CONTRIBUTING.md

```diff
@@ -166,7 +166,7 @@ For more information about `Pull Request` creation please visit [https://docs.gi
 ## Naming Convention
 
 For this project, naming convention for all language use in this project should be on lower camelCase,
-except for Classes, enum types, typedefs, and type parameters these one should be on Upper CamelCase (PascalCase)
+except for Classes, enum types, typedefs, and type parameters these one should be on kebab-case.
```

Comment:
Classes, enum type, typedefs, and type parameters should be on PascalCase

Components references in Yaki Admin (VueJS) should be on  kebab-case